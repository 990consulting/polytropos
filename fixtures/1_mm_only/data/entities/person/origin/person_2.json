{
  "invariant": {
    "first_name": "Jane",
    "color_info": {
      "favorite_color": "blue"
    }
  },
  "2010": {
    "weight_in_pounds": 120.6
  },
  "2011": {
    "weight_in_pounds": 126.2
  },
  "2012": {
    "weight_in_pounds": 119.0
  },
  "2013": {
    "weight_in_pounds": 123.1
  }
}
