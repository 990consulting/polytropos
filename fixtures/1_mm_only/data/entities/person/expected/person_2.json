{
  "invariant": {
    "first_name": "Jane",
    "gender": "female",
    "total_weight_gain": 2.5,
    "personal_summary": "Jane's favorite color is blue (0000FF). Over the observation period, she gained 2.5 lbs.",
    "color_info": {
      "favorite_color": "blue",
      "rgb_value": "0000FF"
    }
  },
  "2010": {
    "weight_in_pounds": 120.6
  },
  "2011": {
    "weight_in_pounds": 126.2
  },
  "2012": {
    "weight_in_pounds": 119.0
  },
  "2013": {
    "weight_in_pounds": 123.1
  }
}
